dist/js/slider/slider.js

```diff
@@ -0,0 +1,31 @@
+export default class Slider {
+    constructor(element) {
+        this.firstSliderControlsNode = element.children[0];
+        this.firstSliderContentNode = element;
+        this.firstSliderControls = Array.from(this.firstSliderControlsNode.children);
+        this.firstSliderContent = Array.from(this.firstSliderContentNode.children);
+    }
+
+    addSliderEventListeners() {
+        this.firstSliderControlsNode.addEventListener('click', (event) => {
+            if (!event.target.classList.contains('controls__control')) {
+                return;
+            }
+
+            const selectedIndex = this.firstSliderControls.indexOf(event.target);
+
+            this.firstSliderControls.forEach((item) => {
+                item.classList.remove('selected__control');
```

Comment:
Looks like the class `selected__control` is used as a modifier for some other class... let's rename it by starting with `_`

Reply:
agree

Reply:
fixed